src/chrome/content/rules/hardwareluxx.ru.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="hardwareluxx.ru">        
```

Comment:
I just realized that adding **platform = "mixedcontent"** breaks the script and it no longer works. I do not have enough knowledge to fix it.